app/components/upload-image.hbs

```diff
@@ -0,0 +1,61 @@
+<div class="image-upload">
+  <div
+    class="drop-area {{if this.overDropZone 'drop-area__highlight'}}"
+    {{on 'drop' this.handleDrop}}
+    {{on 'dragover' this.handleDragOver}}
+    {{on 'dragenter' this.handleDragEnter}}
+    {{on 'dragleave' this.handleDragLeave}}
+  >
+    <form class="image-form">
+      <p class="image-form__text">
```

Comment:
Can we nest `<p>` tags?

Reply:
https://www.w3.org/TR/html401/struct/text.html#h-9.3.1